subprojects/packagefiles/glfw/docs/meson.build

```diff
@@ -0,0 +1,7 @@
+###  GLFW documentation
+#@ Build check
+if not get_option('build-docs')
+  subdir_done()
+endif
+
+warning('Documentation build is not implemented yet!')
```

Comment:
What's the point of having a build option for enabling a feature that is not implemented? Anyhow, it would make more sense to move the conditional to the main `meson.build` and make `subdir(...)` the body of the conditional.

Reply:
> Anyhow, it would make more sense to move the conditional to the main `meson.build` and make `subdir(...)` the body of the conditional.

Is it a necessary change? I suppose it saves time, and I have no good argument other than "I preferred to keep my 'subdir(...)' logic inside their own build files."